GliaWidgetsTests/CallVisualizer/VideoCall/VideoCallTests.swift

```diff
@@ -12,21 +12,27 @@ final class VideoCallTests: XCTestCase {
     }
 
     func test_end_screen_sharing() {
-        let viewModel: CallVisualizer.VideoCallViewModel = .mock()
-        viewModel.environment.screenShareHandler.status.value = .started
+        let screenShareHandlerMock = ScreenShareHandler.live
```

Comment:
As you can see you have to use `live` is tests, and this uses singleton in tests, even though `ScreenShareHandler` is a struct but Observables there are classes. Meaning that they will be affected by other tests. This is what is mentioning earlier.

Reply:
c17663b6a91b022e075a6c811e0fefa5a9ae7420